scripts/pcm-dpc/covid-19/preprocess.py

```diff
@@ -0,0 +1,180 @@
+                    'geoNode': geo_node}))
+                idx += 1
+
+    def _translate(self):
+        """translate coloumn names from italian to english"""
+        it2en = {'data': 'Date', 'stato': 'State',
+                 'codice_regione':'RegionCode',
+                 'denominazione_regione': 'RegionName',
+                 'codice_provincia': 'ProvinceCode',
+                 'denominazione_provincia': 'ProvinceName',
+                 'sigla_provincia': 'ProvinceAbbreviation',
+                 'lat': 'Latitude', 'long': 'Longitude',
+                 'ricoverati_con_sintomi': 'HospitalizedsWithSymptoms',
+                 'terapia_intensiva': 'IntensiveCare',
+                 'totale_ospedalizzati': 'Hospitalized',
+                 'isolamento_domiciliare': 'PeopleInHomeIsolation',
+                 'totale_positivi': 'ActiveCase',
+                 'variazione_totale_positivi': 'IncrementalPositiveCase',
+                 'nuovi_positivi': 'IncrementalActiveCase',
+                 'dimessi_guariti': 'CumulativeRecovered',
+                 'deceduti': 'CumulativeDeath',
+                 'totale_casi':'CumulativePositiveCase',
+                 'tamponi': 'CumulativeTestsPerformed',
+                 'casi_testati': 'CumulativeTestedPeople',
+                 'casi_da_sospetto_diagnostico': 'PositiveCasesFromClinicActivity',
+                 'casi_da_screening': 'PositiveCasesFromSurveyAndTest'}
+        self.data = self.data.rename(columns=it2en)
+
+    def set_location(self):
+        pass
+
+    def geo_template(self):
+        pass
+
+class PcmDpcNational(PcmDpc):
+    """subclass processing national data"""
+    def __init__(self):
+        self.csvpath = ('https://raw.githubusercontent.com/pcm-dpc/COVID-19/'
+                        'master/dati-andamento-nazionale/dpc-covid19-ita-and'
+                        'amento-nazionale.csv')
+        self.name = "dpc-covid19-ita-national-trend"
+        self.stat_vars = ['HospitalizedsWithSymptoms', 'IntensiveCare',
+                          'Hospitalized', 'PeopleInHomeIsolation', 'ActiveCase',
+                          'IncrementalPositiveCase', 'IncrementalActiveCase',
+                          'CumulativeRecovered', 'CumulativeDeath',
+                          'CumulativePositiveCase', 'CumulativeTestsPerformed',
+                          'CumulativeTestedPeople']
+
+    def set_location(self):
+        assert (self.data['State'] == 'ITA').all()
+        self.data = self.data.drop(columns=['State'])
+
+    def geo_template(self):
+        return 'dcid:country/ITA'
+
+
+class PcmDpcRegions(PcmDpc):
+    """subclass processing regional data"""
+    def __init__(self):
+        self.csvpath = ('https://raw.githubusercontent.com/pcm-dpc/COVID-19/'
+                        'master/dati-regioni/dpc-covid19-ita-regioni.csv')
+        self.name = "dpc-covid19-ita-regional"
+        self.stat_vars = ['HospitalizedsWithSymptoms', 'IntensiveCare',
```

Comment:
Since the columns are repeated, you could pull out the national and regional set into a _SOME_VARIABLE and assign self.stat_vars it in the two classes.

Reply:
done.